Doc: image examples

sugetha24 opened the pull request:
## Change Summary

added image examples

## How to test
`ansible-playbook examples/<name-of-example>.yaml`

## Checklist

### Repository Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

sugetha24 committed:
Doc: image examples

Vibhu-gslab commented on ansible_collections/arista/cvp/examples/cv_image_v3/bundle_delete.yaml:
```diff
@@ -0,0 +1,26 @@
+  gather_facts: no
+  vars:
+  tasks:
+    - name: "Update an image {{inventory_hostname}}"
```
This would be Remove bundle

Vibhu-gslab commented on ansible_collections/arista/cvp/examples/cv_image_v3/bundle_upload.yaml:
```diff
@@ -0,0 +1,26 @@
+  gather_facts: no
+  vars:
+  tasks:
+    - name: "Update an image {{inventory_hostname}}"
```
This would be Create a bundle or Add a bundle

Vibhu-gslab commented on ansible_collections/arista/cvp/examples/cv_image_v3/image_delete.yaml:
```diff
@@ -0,0 +1,25 @@
+  vars:
+  tasks:
+    # API doesn't support image delete! Should throw error message
+    - name: "Update an image {{inventory_hostname}}"
```
This would be Delete an image

Vibhu-gslab commented on ansible_collections/arista/cvp/examples/cv_image_v3/image_delete.yaml:
```diff
@@ -0,0 +1,25 @@
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: image
+        action: delete
```
```suggestion
        action: remove
```

Vibhu-gslab commented on ansible_collections/arista/cvp/examples/cv_image_v3/image_upload.yaml:
```diff
@@ -0,0 +1,24 @@
+  gather_facts: no
+  vars:
+  tasks:
+    - name: "Update an image {{inventory_hostname}}"
```
This would be Add an image or Upload an image

sugetha24 committed:
typo fix

noredistribution committed:
Update bundle_delete.yaml

removing image_list as it's not necessary when deleting the bundle

noredistribution committed:
Rename bundle_upload.yaml to bundle_create.yaml

renaming to bundle_create as we don't upload the bundle but create it from uploaded images

noredistribution reviewed (approved):
LGTM!

Shivani-chourasiya reviewed (approved):
LGTM!

Vibhu-gslab reviewed (approved):
LGTM!
